tests/rest/admin/test_device.py

```diff
@@ -410,13 +459,21 @@ def test_get_devices(self) -> None:
         self.assertEqual(number_devices, channel.json_body["total"])
         self.assertEqual(number_devices, len(channel.json_body["devices"]))
         self.assertEqual(self.other_user, channel.json_body["devices"][0]["user_id"])
-        # Check that all fields are available
+        # Check that all fields are available, and that the dehydrated device is marked as dehydrated
+        found_dehydrated = False
         for d in channel.json_body["devices"]:
             self.assertIn("user_id", d)
             self.assertIn("device_id", d)
             self.assertIn("display_name", d)
             self.assertIn("last_seen_ip", d)
             self.assertIn("last_seen_ts", d)
+            if d["device_id"] == "dehydrated_device":
+                self.assertEqual(True, d.get("dehydrated"))
+                found_dehydrated = True
+            else:
+                self.assertNotIn("dehydrated", d)
```

Comment:
To align with the suggestion above.

```suggestion
                # Either the field is not present, or set to False
                self.assertFalse(d["dehydrated])
```